### PR TITLE
ppbt-40_response_status_for_createUser

### PR DESCRIPTION
**Pull request by aleksdenni**
Fixes the issue of creating a user with a duplicate email.

### CODE REVIEW

**Review by alex97po — CHANGES_REQUESTED** [review]
Remove unnecessary RuntimeException

File: pom.xml
Comment (aleksdenni) [inline pom.xml:1]: Deletes the code duplicate. The code in line 52 is identical

File: src/main/java/com/example/java/test/junior/developer/keycloak/KeycloakAdminClient.java [hunk 1]
```diff
@@ -16,7 +19,17 @@ public class KeycloakAdminClient {
...
-    keycloak.realm(keycloakProperties.getRealm()).users().create(user);
-    log.info("User created successfully");
+    var response = keycloak.realm(keycloakProperties.getRealm()).users().create(user);
+    int statusCode = response.getStatus();
+    if (statusCode == Response.Status.CREATED.getStatusCode()) {
+      log.info("User created successfully with status code: {}", statusCode);
+    } else {
+      log.error("User creation failed with status code: {}", statusCode);
+      try {
+        throw new RuntimeException("User creation failed");
```
Comment (alex97po) [inline src/main/java/com/example/java/test/junior/developer/keycloak/KeycloakAdminClient.java]: What is the purpose of this line? 
You throw exception then immediately clutch it on the next line. It doesn't make any sense...

Comment (aleksdenni) [reply src/main/java/com/example/java/test/junior/developer/keycloak/KeycloakAdminClient.java:29]: Yes, it doesn't make sense now. Just a remnant of previously written code. I don't know why I didn't delete it.

**Review by NatMyronenko — APPROVED** [review]

**Review by NatMyronenko — APPROVED** [review]

**Review by alex97po — APPROVED** [review]

**Pull request merged**